app/code/Magento/AdminAnalytics/Test/Unit/Condition/CanViewNotificationTest.php

```diff
@@ -21,31 +20,28 @@ class CanViewNotificationTest extends TestCase
     /** @var CanViewNotification */
     private $canViewNotification;
 
-    /** @var  Logger|MockObject */
+    /** @var Logger|MockObject */
     private $viewerLoggerMock;
 
-    /** @var ProductMetadataInterface|MockObject */
-    private $productMetadataMock;
+    /** @var CacheInterface|MockObject */
+    private $cacheStorageMock;
 
-    /** @var  Log|MockObject */
+    /** @var Log|MockObject */
     private $logMock;
 
-    /** @var MockObject|CacheInterface $cacheStorageMock */
-    private $cacheStorageMock;
-
     protected function setUp(): void
     {
+        $this->viewerLoggerMock = $this->createMock(Logger::class);
         $this->cacheStorageMock = $this->getMockBuilder(CacheInterface::class)
             ->getMockForAbstractClass();
+
         $this->logMock = $this->createMock(Log::class);
-        $this->viewerLoggerMock = $this->createMock(Logger::class);
-        $this->productMetadataMock = $this->getMockForAbstractClass(ProductMetadataInterface::class);
+
         $objectManager = new ObjectManager($this);
         $this->canViewNotification = $objectManager->getObject(
             CanViewNotification::class,
             [
                 'viewerLogger' => $this->viewerLoggerMock,
-                'productMetadata' => $this->productMetadataMock,
```

Comment:
We don't have `productMetadata` at Constructor 
https://github.com/magento/magento2/blob/01c167ade67eabb071d8975bdce2aec93dd86933/app/code/Magento/AdminAnalytics/Model/Condition/CanViewNotification.php#L47